modules/machine-health-checks-about.adoc

```diff
@@ -9,13 +9,13 @@
 
 Machine health checks automatically repair unhealthy machines in a particular machine pool.
 
-To monitor machine health, create a resource to define the configuration for a controller. Set a condition to check, such as staying in the `NotReady` status for five minutes or displaying a permanent condition in the node-problem-detector, and a label for the set of machines to monitor.
-
 [NOTE]
 ====
-You cannot apply a machine health check to a machine with the master role.
+You can only apply a machine health check to control plane machines on clusters that use control plane machine sets.
```

Comment:
@JoelSpeed moved the note up a paragraph (think it makes more sense there) and rephrased.